Return esphome cover position as Integer

neffs opened the pull request:
## Description: 
Cover position is [specified as Integer 0-100](https://www.home-assistant.io/components/cover/), we should not return float here.

**Related issue (if applicable):** fixes #25738

## Checklist:
  - [X] The code change is tested and works locally.
  - [X] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [X] There is no commented out code in this PR.
  - [X] I have followed the [development checklist][dev-checklist]

MartinHjelmare commented on homeassistant/components/esphome/cover.py:
```diff
@@ -91,11 +91,11 @@ def is_closing(self) -> bool:
         if not self._static_info.supports_position:
             return None
-        return self._state.position * 100.0
+        return int(round(self._state.position * 100.0))
```
`round` will return an integer if the second argument is omitted. We can remove the extra copy to integer.

neffs committed:
output esphome cover position as integer

cover position is specified as integer 0-100, we should not return
float here.
fixes #25738

MartinHjelmare reviewed (approved):
Thanks!
